amads/algorithms/scale.py

```diff
@@ -2,21 +2,21 @@
 
 Original doc: https://citeseerx.ist.psu.edu/document?repid=rep1&type=pdf&doi=6e06906ca1ba0bf0ac8f2cb1a929f3be95eeadfa#page=88
 
-Scales note data in given dimension (start, duration, or both)
+Scales note data in given dimension (onset, duration, or both)
 
 Input arguments:
 SCORE = score object/other EventGroup object (Note: This method MODIFIES the score object input)
 FACTOR = amount of scale (must be > 0)
-DIM = dimension ('start', 'duration','all')
-    - 'start': scales the start of ALL EVENTS
+DIM = dimension ('onset', 'duration','all')
+    - 'onset': scales the onset of ALL EVENTS
     - 'duration': scales the durations of ALL NON-EVENTGROUP EVENTS (Note, Rest)
-    - 'all': scales both the start and the duration
+    - 'all': scales both the onset and the duration
 
 Output:
 SCORE = the scaled version of the input object
 
 Examples:
-scaled_score = scale(score.copy(),'start',2); % scales start by a factor of 2
+scaled_score = scale(score.copy(),'onset',2); % scales onset by a factor of 2
```

Comment:
Shouldn't this make a deep copy? (I'm in the process of changing copy/deepcopy in another branch, so I'm fine to leave this as is and fix it elsewhere, but can we specify that the implementation will not modify score?)